docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +11,16 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-0-1]]
+=== Logstash 6.0.1 Release Notes
+* Elasticsearch Output: Ignore event's type field for the purpose of setting document _type if cluster is es 6.x or above
```

Comment:
Change to:

Elasticsearch Output: We now ignore an event's type field for the purpose of setting the document _type if the cluster is Elasticsearch 6.x or above.